clients/python/anna/lattices.py

```
@@ -150,6 +150,32 @@ def serialize(self):
+        """Return fragments concatenated in sorted order as bytes."""
+        fragments = sorted(
+            v.decode("utf-8", errors="replace") if isinstance(v, bytes) else str(v)
+            for v in self.val
+        )
+        return "\n".join(fragments).encode("utf-8")
+
+    def serialize(self):
+        from .kvs_pb2 import UNION_SCALAR as US
+        res = SetValue()
+        for v in self.val:
+            if type(v) != bytes:
```

Ruff (0.16.0)

[error] Use `is` and `is not` for type comparisons, or `isinstance()` for isinstance checks

(E721)